Document/0x05a-Platform-Overview.md

```diff
@@ -127,9 +127,8 @@ drwxrwx--x u0_a65   u0_a65            2016-01-10 09:44 shared_prefs
 
 #### APK Structure
 
-An app on Android is a file with the extension .apk. This file is a signed zip-file which contains all of the application's resources, byte code, etc. When unzipped the following directory structure can usually be identified:
+An app on Android is a file with the extension .apk. This file is a signed zip-file which contains all the application's resources, byte code, etc. When unzipped the following directory structure can usually be identified:
 
-```bash

```

Comment:
If you remove this it is no longer formatted as code, which means that the newlines are ignored and everything ends up on one line.